pkg/printers/internalversion/describe_test.go

```diff
@@ -118,6 +118,28 @@ func TestDescribePodTolerations(t *testing.T) {
 	}
 }
 
+func TestDescribeSecret(t *testing.T) {
+	fake := fake.NewSimpleClientset(&api.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "bar",
+			Namespace: "foo",
+		},
+		Data: map[string][]byte{
+			"username": []byte("YWRtaW4="),
+			"password": []byte("MWYyZDFlMmU2N2Rm"),
+		},
+	})
+	c := &describeClient{T: t, Namespace: "foo", Interface: fake}
+	d := SecretDescriber{c}
+	out, err := d.Describe("foo", "bar", printers.DescriberSettings{})
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	if !strings.Contains(out, "bar") || !strings.Contains(out, "foo") || !strings.Contains(out, "username") || !strings.Contains(out, "8 bytes") || !strings.Contains(out, "password") || !strings.Contains(out, "16 bytes") {
```

Comment:
Yes, explicitly check for the presence of the sensitive data that's not supposed to be present on the output, and error out if it's there.